tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinatorHelpers.Core.cs

```diff
@@ -78,7 +80,17 @@ internal static void CheckPathParams(this Security security, HttpContext context
             {
                 var securityCoordinator = SecurityCoordinator.Get(security, span, transport);
                 var args = new Dictionary<string, object> { { AddressesConstants.RequestPathParams, pathParams } };
-                var result = securityCoordinator.RunWaf(args);
+                IResult? result;
+                // we need these tests for netcoreapp3.1 and lower, as otherwise it throws
```

Comment:
There's nothing netcore31+ specific AFAICT, so it doesn't matter, but why only those TFMs? Don't we need the check for newer versions too? 🤔

Reply:
I added the check for all versions. Funny enough, it was crashing only on older versions in integration tests so I wrote this comment. But now browsing the source code I can see that even the latest version does:
```csharp
   public override ISession Session
    {
        get
        {
            var feature = SessionFeatureOrNull;
            if (feature == null)
            {
                throw new InvalidOperationException("Session has not been configured for this application " +
                    "or request.");
            }
            return feature.Session;
        }
```
So I'll remove the comment in af317c569022b4987e06b32b3f0358e884eb31f5 regarding netcoreapp3.1 and lower !